Static/Section.swift

```diff
@@ -43,6 +43,9 @@ public struct Section: Hashable, Equatable {
 
     // MARK: - Properties
 
+    /// The section's accessibility identifier.
+    public var accessibilityIdentifier: String?
```

Comment:
Where is this variable used? It looks like we assign an accessibility identifier to a each cell, but this one for the whole section looks unused.

Reply:
Yeah good call out - it is unused. I've been trying to figure out how to apply this identifier to the section header/footer but am having a bit of trouble trying to implement it. I think for now I'll leave this off and revisit it later.